server.js

```diff
@@ -38,7 +38,7 @@ function renderPage(appHtml, reactHelmet) {
         <link rel="shortcut icon" href="/assets/favicons/favicon.ico">
         <link rel="manifest" href="/manifest.json">
         <link rel="stylesheet" type="text/css" href="https://code.cdn.mozilla.net/fonts/zilla-slab.css">
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,300,700,300i">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito+Sans:300,300i,400,600,700">
```

Comment:
Do we need the 300 weights? A command+F doesn't find that being used

Reply:
Really?...I'm using it over in #1059 & it's used in the little capsule when you search a help cat. on the homepage. As for 300i, I guess that can be removed.

Reply:
Ah, that's probably why it didn't show up in a search of _that page_. Obviously they're fine to keep if they're used 👍